typescript-modal/main.ts

```diff
@@ -0,0 +1,13 @@
+const $openModal = document.querySelector('.open-modal');
+const $dismissModal = document.querySelector('.dismiss-modal');
+const $dialog = document.querySelector('dialog');
+
+if (!$openModal || !$dismissModal || !$dialog) {
+  throw new Error('dialog box not found');
+}
+
```

Comment:
This will work to handle the TypeScript red squigglys, but if you were to have a failed query on a specific element then how do you know which element it is? 🤔